### PR TITLE
IA-2800 Upgrade Galaxy to 21.05

### PR DESCRIPTION
**Pull request by rtitle**
https://broadworkbench.atlassian.net/browse/IA-2800

`galaxykubeman` was merged here: https://github.com/galaxyproject/galaxykubeman-helm/pull/31

Will see how automation tests do. Needs a UI PR as well to update the version string.

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by rtitle** [comment]
Cool, automation tests passed but would also like to do some manual tests before merge.

**Comment by rtitle** [comment]
```
org.broadinstitute.dsde.workbench.leonardo.runtimes.RuntimeCreationDiskSpec.create runtime and attach a persistent disk
```
```
message: Failed to delete runtime gpalloc-qa-master-tuvexa7/automation-test-a6y4fqhez, status: 500 Internal Server Error body: {"source":"leonardo","message":"Fail to detach DiskName(etzgenaalt) from RuntimeName(automation-test-a6y4fqhez) in a timely manner","statusCode":500,"exceptionClass":"class java.lang.RuntimeException","traceId":null}
```

jenkins retest

**Comment by rtitle** [comment]
I did some basic testing:
 - Create a Galaxy runtime
 - Verify the `21.05` version in `Help -> About Galaxy`
 - Load some data, run a workflow
 - Delete Galaxy, keep disk, confirm deletion worked
 - Create a second Galaxy, confirm workflow data still on disk
 - Delete the Galaxy and disk, confirm deletion worked

File: http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreterSpec.scala
Comment (jdcanas) [inline http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreterSpec.scala:101]: I hate these tests....

File: Dockerfile
Comment (jdcanas) [inline Dockerfile:34]: huh... whats the correlation between this number and the one in terra-UI (21.05)

**Review by jdcanas — APPROVED** [review]

File: Dockerfile
Comment (rtitle) [reply Dockerfile:34]: This is technically the `galaxykubeman` helm chart version, which references the Galaxy _app_ version 21.05..

**Comment by rtitle** [comment]
Posted on slack, just going to wait on Galaxy team confirmation before merge.

**Review by Qi77Qi — APPROVED** [review]

File: Dockerfile
Comment (Qi77Qi) [reply Dockerfile:34]: would be nice to add the galaxy app version as a comment

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
got the go ahead from anvil for this

**Comment by rtitle** [comment]
jenkins retest

**Pull request merged**
